hack/e2e-common.sh

```diff
@@ -72,13 +77,47 @@ function install_jobset {
     kubectl apply --server-side -f "${JOBSET_MANIFEST}"
 }
 
+function patch_kubeflow_manifest {
+    # In order for MPI-operator and Training-operator to work on the same cluster it is required that:
+    # 1. 'kubeflow.org_mpijobs.yaml' is removed from base/crds/kustomization.yaml - https://github.com/kubeflow/training-operator/issues/1930
+    sed -i '/kubeflow.org_mpijobs.yaml/d' "${KUBEFLOW_CRDS}/base/crds/kustomization.yaml"
+
+    # 2. Training-operator deployment file is patched and manually enabled for all kubeflow jobs except for mpi -  https://github.com/kubeflow/training-operator/issues/1777
+    KUBEFLOW_DEPLOYMENT="${KUBEFLOW_CRDS}/base/deployment.yaml"
+
+    # Find the line after which to insert the args
+    INSERT_LINE=$(grep -n "^ *- /manager" "${KUBEFLOW_DEPLOYMENT}" | head -n 1 | cut -d ':' -f 1)
+
+    # Prepare patch with the args after the specified line
+    # EOF must be indented most left - doesn't work otherwise
+    ARGS_PATCH=$(cat <<EOF
+          args:
+            - --enable-scheme=tfjob
+            - --enable-scheme=pytorchjob
+            - --enable-scheme=xgboostjob
+            - --enable-scheme=paddlejob
+EOF
+)
+    # Apply the patch
+    sed -i -e "${INSERT_LINE}r /dev/stdin" "${KUBEFLOW_DEPLOYMENT}" << EOF
```

Comment:
just a matter of preference it would be something like:
`yq eval 'del(.resources[] | select(. == "kubeflow.org_mpijobs.yaml"))' $(EXTERNAL_CRDS_DIR)/training-operator/base/crds/kustomization.yaml`

Reply:
it's already copied, I mean it's recommended to apply such patches